pkg/operator/platform/platform.go

```diff
@@ -59,3 +67,55 @@ func getClient(explicitKubeconfig string) (client.Client, error) {
 
 	return dynamicClient, nil
 }
+
+func GetFeatureGates() (featuregates.FeatureGate, error) {
+	stop := make(chan struct{})
+	ctx := context.Background()
```

Comment:
The code in the command's initialization creates a context and binds it to signals, leader election, etc - pefer to pass that in here?

Reply:
Okay passing in context now in an upcoming commit.

Can you determine/explain if/why we should keep that `context.WithCancel(ctx)` call and subsequent goroutine that immediately follows?